File: api/python/examples/sub.py
#encoding=utf8
from zbus import SingleBroker, Consumer, MqMode

broker = SingleBroker(host='localhost', port=15555)

consumer = Consumer(broker=broker,mq='MyPubSub',mode=MqMode.PubSub) 

consumer.topic = 'sse' #指定感兴趣的消息主题，用','分割不同主题

while True:
    msg = consumer.recv()
    if msg is None: continue
    print msg
     

broker.destroy()  

File: api/nodejs/examples/sub.js
var zbus = require("../zbus");
var Message = zbus.Message;
var MessageClient = zbus.MessageClient;
var Consumer = zbus.Consumer;
var MqAdmin = zbus.MqAdmin;
var Mode = zbus.MqMode;


var client = new MessageClient("127.0.0.1:15555");
client.connect(function(){
	 var c = new Consumer(client, "MyPubSub");
     c.mode = Mode.PubSub; 
	 c.topic = 'sse';
	 
     c.recv(function(msg){
    	 console.log(msg.toString());
     });
});
